### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by melisayu** [alexrobaina/frontend_petsLove#260] [General]
Hi, is this project still active? I would like to contribute. I can't access the link though.

**Comment by alexrobaina** [comment]
Hi @melisayu 😄 
Yes! This project is active! 

Thanks for your interest in Pet's Love.

Please check these links: 
https://landing.petslove.app/

This is the app in prod: 
https://petslove.app/ 

This is beta: 
https://beta.petslove.app/

What do you want to do? Do you like frontend or backend?

**Reply by melisayu** [reply]
Hi Alex, thanks for your reply.
I'm a fullstack but I will start with frontend.

**Reply by alexrobaina** [reply]
Okay, nice! 
If you have any questions about how to run the project in your local, please feel free to ask.

**Comment by alexrobaina** [comment]
Hi @melisayu! I'm sorry for the delay. 
If you want, you can start with the following issue: 
https://github.com/alexrobaina/frontend_petsLove/issues/261

It is good to start. 😄 

Thank you

**Reply by melisayu** [reply]
Yes I want that. Could you please assign it to me?

**Reply by alexrobaina** [reply]
Yes, of course. Can you please comment on something on the issue? I can't assign Jet.
